src/query/query_manager.c

```diff
@@ -1281,6 +1279,8 @@ xqmgr_execute_query (THREAD_ENTRY * thread_p, const XASL_ID * xasl_id_p, QUERY_I
   bool is_xasl_pinned_reference;
   bool do_not_cache = false;
 
+  static int qmgr_max_query_entry_per_tran = prm_get_bool_value (PRM_ID_QMGR_MAX_QUERY_PER_TRAN);
```

Comment:
Is prm_get_bool_value() not prm_get_integer_value() ?

Reply:
fixed. thanks.